_includes/js/queries.md

```diff
@@ -205,6 +205,39 @@ The above example will match any `BarbecueSauce` objects where the value in the
 
 Queries that have regular expression constraints are very expensive, especially for classes with over 100,000 records. Parse restricts how many such operations can be run on a particular app at any given time.
 
+### Full Text Search
+
+Use `fullText` for efficient search capabilities. Text indexes are automatically created for you. Your strings are turned into tokens for fast searching.
+
+* Note: Full Text Search can be resource expensive. As with all indexes ensure you have enough RAM. Speed comes at a cost.
+
+Requires Parse Server 2.5.0+
```

Comment:
I was wondering about these requirement labels, we should try to standardize them so they're not just dotted around. I think a solution could be to make sure this immediately follows the feature title. In this case `### Full Text Search`. Also, and this is just a thought, but maybe we should italicize this entire requirement, just to emphasize it before the reader attempts to start implementing said feature.